src/flamegraph/flamegraph.js

```diff
@@ -143,6 +185,11 @@ function update_text(e) {
     }
     t.textContent = "";
 }
+function update_search_elements() {
```

Comment:
I may be missing something, but this doesn't seem to be called?

Reply:
You're right, I moved the body of this into `update_for_width_change` above and forgot to delete it.